Bottom: 1f11c9dab026e8204a7abce2a87d04a9c9e5aaea
Top:    6e9cbbc3640934c94f15a7079574fb9b9380c3d0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-12 23:31:28 +0000

Refresh of hidgl-fixup-to-use-hid_draw

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 1f790d2..f611005 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2589,7 +2589,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
 
   ghid_draw_bg_image ();
 
-  common_set_clip_box (hid_draw, &region);
+  common_set_clip_box (priv->hid_draw, &region);
   /* hid_expose_callback (&ghid_graphics, &region, 0); */
   ghid_draw_everything (&region);
   hidgl_flush_triangles (priv->hid_draw);
@@ -3009,7 +3009,6 @@ HID_DRAW *
 ghid_request_debug_draw (void)
 {
   GHidPort *port = gport;
-  render_priv *priv = port->render_priv;
   GtkWidget *widget = port->drawing_area;
   GtkAllocation allocation;
